{"name": "test_non_existen_courier_login_message", "status": "failed", "statusDetails": {"message": "AssertionError: assert '{\"code\":404,... не найдена\"}' == '{\"message\": ... не найдена\"}'\n  \n  - {\"message\": \"Учетная запись не найдена\"}\n  ?            -\n  + {\"code\":404,\"message\":\"Учетная запись не найдена\"}\n  ?  +++++++++++", "trace": "self = <tests.test_courier_login.TestCourierLogin object at 0x00000245D1F52720>\n\n    @allure.description('Проверка тела ответа при передаче логина и пароля несуществующего курьера в теле запроса логина курьера')\n    def test_non_existen_courier_login_message(self):\n        courier_login_request = ScooterApi.courier_login(DataBodies.NON_EXISTEN_COURIER)\n>       assert courier_login_request.text == '{\"message\": \"Учетная запись не найдена\"}'\nE       assert '{\"code\":404,... не найдена\"}' == '{\"message\": ... не найдена\"}'\nE         \nE         - {\"message\": \"Учетная запись не найдена\"}\nE         ?            -\nE         + {\"code\":404,\"message\":\"Учетная запись не найдена\"}\nE         ?  +++++++++++\n\ntests\\test_courier_login.py:57: AssertionError"}, "description": "Проверка тела ответа при передаче логина и пароля несуществующего курьера в теле запроса логина курьера", "start": 1725817751294, "stop": 1725817751578, "uuid": "9fe3c757-0598-4ad6-b0c5-8f0b45ad16f6", "historyId": "af8acaab08e4483a1a017672561af0ec", "testCaseId": "af8acaab08e4483a1a017672561af0ec", "fullName": "tests.test_courier_login.TestCourierLogin#test_non_existen_courier_login_message", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "mr_cat"}, {"name": "thread", "value": "17924-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}